templates/javascript-ethers/scripts/deploy.js

```diff
@@ -32,12 +33,12 @@ async function main() {
     await rrpBeaconServerMock.deployed();
     rrpBeaconServerAddress = rrpBeaconServerMock.address;
   } else {
-    // TODO: replace with path to services repo
-    const deployments = require(`../services/data/beacons/0.3.1/${network}.json`);
-    rrpBeaconServerAddress =
-      deployments.contracts[
-        "@api3/airnode-protocol/contracts/rrp/requesters/RrpBeaconServer"
-      ];
+    const deployments = getServiceData(
+      "Amberdata",
```

Comment:
This needs to change for the indefinitely running beacon, but I am not sure what's the progress there.

Reply:
Decided on call, we will use amberdata which are available till April.